packages/common/.npmignore

```diff
@@ -39,3 +39,5 @@ lib/**/injected/
 !browsers.json
 # Include commit info
 !commitinfo
+# Include generated devices descriptors
+deviceDescriptorsSource.json
```

Comment:
```suggestion
!deviceDescriptorsSource.json
```

Reply:
You are faster than the bots report to me! :p